src/llscan.cc

```diff
@@ -101,7 +101,20 @@ bool FindInstancesCmd::DoExecute(SBDebugger d, char** cmd,
 
   inspect_options.detailed = detailed_;
 
-  char** start = ParseInspectOptions(cmd, &inspect_options);
+  char** start = cmd;
+  for (; cmd != nullptr && *start != nullptr; start++) {
+    if (strcmp(*start, "-i") == 0 || strcmp(*start, "--inspect") == 0) {
+      inspect_options.detailed = true;
+      // Shuffle up the remaining parameters.
+      char** curr = start;
+      for (; *curr != nullptr; curr++) {
+        char** next = curr + 1;
+        *curr = *next;
+      }
+    }
+  }
```

Comment:
IMO, this ad hoc option parser is kind of nasty.  Is there a reason you can't extend `ParseInspectOptions()`?